### PR TITLE
https://github.com/adafruit/Adafruit_CircuitPython_SimpleIO/issues/10

### PR DESCRIPTION
**Pull request by apatt**
I was able to test that the parameter passing works as expected and that there are not python errors, but I don't know to confirm the underlying functions (switch_to_input and switch_to_output) functions work as expected.

### CODE REVIEW

File: simpleio.py [hunk 1]
```diff
@@ -230,9 +230,14 @@ class DigitalOut:
     """
     Simple digital output that is valid until soft reset.
     """
-    def __init__(self, pin):
+    def __init__(self, pin, value=None, drive_mode=None):
```
Comment (tannewt) [inline simpleio.py]: Simpler to do `def __init__(self, pin, **kwargs):`

File: simpleio.py [hunk 2]
```diff
@@ -230,9 +230,14 @@ class DigitalOut:
...
+        """
+        value and drive_mode: if passed in will be passed to switch_to_output
+            see digitalio.DigitalInOut.switch_to_output for more details
```
Comment (tannewt) [inline simpleio.py]: Please copy the docs here. Directing elsewhere doesn't simplify the docs.

**Review by tannewt — CHANGES_REQUESTED** [review]
THanks for picking this up. A couple comments to simplify things.

**Comment by apatt** [comment]
ok, fixed all of your comments.  again, tested to make sure python doesn't fail.

**Review by tannewt — APPROVED** [review]

**Comment by tannewt** [comment]
Thanks @apatt !

**Pull request merged**